src/test/java/org/tdl/vireo/controller/SubmissionControllerTest.java

```diff
@@ -0,0 +1,96 @@
+package org.tdl.vireo.controller;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+import edu.tamu.weaver.response.ApiResponse;
+import edu.tamu.weaver.response.ApiStatus;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.tdl.vireo.model.ActionLog;
+import org.tdl.vireo.model.Role;
+import org.tdl.vireo.model.SubmissionState;
+import org.tdl.vireo.model.SubmissionStatus;
+import org.tdl.vireo.model.User;
+import org.tdl.vireo.model.repo.ActionLogRepo;
+
+@ActiveProfiles("test")
+@ExtendWith(SpringExtension.class)
+@ExtendWith(MockitoExtension.class)
+public class SubmissionControllerTest {
+
+    private static final String TEST_USER_1_EMAIL = "User 1 email";
```

Comment:
Move to a JSON file as per best practice.

Reply:
I'm going with the existing design practices by Vireo.
I'm not out to change the practices.
If there is a change then we need to do it across the entire Vireo project.
I suggest creating an issue for changing the test code practices.